docs/source/index.rst

```diff
@@ -57,25 +34,38 @@ Citing this work
    journal={bioRxiv},
```

Comment:
in the docs it appears in bibtex format. Is that what we want?

Reply:
Yes, I meant this as a bibtex entry that people could directly copy-paste :). Alternatively we could provide a download link for the bibtex entry, or direct to another page